app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTestV2.kt

```diff
@@ -482,4 +482,66 @@ class BloodPressureEntrySheetControllerTestV2 {
     verify(bloodPressureRepository, never()).saveMeasurement(any(), any(), any(), any())
     verify(sheet).setBpSavedResultAndFinish()
   }
+
+  @Test
+  @Suppress("IMPLICIT_CAST_TO_ANY")
+  @Parameters(method = "params for showing date validation errors")
+  fun `when save is clicked, date entry is active and input is invalid then validation errors should be shown`(
+      openAs: OpenAs,
+      errorResult: DateOfBirthFormatValidator.Result2,
+      errorUiChangeVerifications: UiChange
```

Comment:
Since this is not exactly a `UiChange`, can we explicitly declare the type as `(BloodPressureEntrySheet) -> Unit` instead?

Reply:
Done